### PR TITLE
feat: update url based on language

### PR DESCRIPTION
**Pull request by mayuran-deriv**

### CODE REVIEW

**Review by eduard-deriv — APPROVED** [review]

**Review by ali-hosseini-deriv — APPROVED** [review]

File: src/js/footer/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 // All the footer js files should be imported and exported from here
 import './helper';
+import './updateUrlAsPerLanguage'
 import './footer/index.js';
```
Comment (ali-hosseini-deriv) [inline src/js/footer/index.js:4]: @mayuran-deriv there is no such file exists. 
please check this.
```suggestion
```

**Review by ali-hosseini-deriv — CHANGES_REQUESTED** [review]

File: src/js/footer/index.js (hunk 1 above)
Comment (mayuran-deriv) [reply src/js/footer/index.js:4]: hey ako src/js/footer/updateUrlAsPerLanguage/index.js 
there is a file here

Comment (ali-hosseini-deriv) [reply src/js/footer/index.js:4]: this is already in footer/index.js file

Comment (mayuran-deriv) [reply src/js/footer/index.js:4]: hey @ali-hosseini-deriv there is a file 
https://github.com/deriv-com/webflow-deriv-com-scripts/blob/master/src/js/footer/footer/index.js

its eduard's code

Comment (ali-hosseini-deriv) [reply src/js/footer/index.js:4]: I see, that's the thing that I didn't notice on the other pr. it was like inception to me :-D 
let me merge this 
will ask @eduard-deriv to rename that directory

**Pull request merged**